### PR TITLE
helm chart

### PR DESCRIPTION
**Pull request by alexisbssn**
helm chart to deploy squidex with mongo in one command

I needed this for myself and I figured I could share it

### CODE REVIEW

File: standalone/kubernetes/helm/squidex/requirements.lock [hunk 1]
```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: mongodb-replicaset
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  version: 3.9.6
+- name: eventstore
```
Comment (SebastianStehle) [inline standalone/kubernetes/helm/squidex/requirements.lock]: Do we need a dependency to event store?

File: standalone/kubernetes/helm/squidex/values.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    # kubernetes.io/tls-acme: "true"
+  hostName: squidex.local
+
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+env:
+  URLS__BASEURL: http://squidex.local/
```
Comment (SebastianStehle) [inline standalone/kubernetes/helm/squidex/values.yaml]: I have never used Helm but should these values not be configurable?

**Comment by SebastianStehle** [comment]
Awesome, I added a few comments.

File: standalone/kubernetes/helm/squidex/requirements.lock (hunk 1 above)
Comment (alexisbssn) [reply standalone/kubernetes/helm/squidex/requirements.lock]: Forgot to remove it. Will do

File: standalone/kubernetes/helm/squidex/values.yaml (hunk 2 above)
Comment (alexisbssn) [reply standalone/kubernetes/helm/squidex/values.yaml]: the way it works is that this whole file exposes default values which you can (should) override.

e.g. We have a dependency on mongo, and the only value I did override was the replica count. (enabled is a value I added) Someone adding Squidex as a dependency on his project would have a squidex section in his values.yaml file to override values

Comment (SebastianStehle) [reply standalone/kubernetes/helm/squidex/values.yaml]: Okay, but I think the default values for the followings fields do not make sense or are dangerous:

* URLS__BASEURL
* LETSENCRYPT_HOST
* LETSENCRYPT_EMAIL
* IDENTITY__ADMINEMAIL
* IDENTITY__ADMINPASSWORD

Can you mark these values as required?

**Comment by alexisbssn** [comment]
Here is an example of what the deployment looks like
image

File: standalone/kubernetes/helm/squidex/values.yaml (hunk 2 above)
Comment (alexisbssn) [reply standalone/kubernetes/helm/squidex/values.yaml]: I believe so. Let me check what I can do

Comment (alexisbssn) [reply standalone/kubernetes/helm/squidex/values.yaml]: Done. The error message goes one at a time and is not the best but it works:

image

And I kept a sample-values file, to use it it's `helm template ./squidex -f sample-values.yaml` from the helm/ folder

With the samples the env variables look like so:
image

**Comment by alexisbssn** [comment]
To have a helm repo (to serve .tgz files instead of having to copy the files around) it might be adviseable to create another repo just for helm. I'm new to this and reading the following: https://medium.com/containerum/how-to-make-and-share-your-own-helm-package-50ae40f6c221

I'll try it out in my fork and let you know

Edit: I tried it. It works fine. Only problem is, I cannot do the required work in this PR, I would need access to the squidex/squidex-docker repo to set it up properly.

So, here are the steps to make it work:
* install helm if you don't have it
* Move to the root path of the repo if not there already and:
* `git checkout -b gh-pages`
* `touch index.yaml`
* `git add index.yaml`
* `git commit -m "adding index.yaml"`
* `git push -u origin gh-pages`
* `helm package standalone/kubernetes/helm/squidex`
* `helm repo index . --url https://squidex.github.io/squidex-docker/`
* git commit & push

The two helm commands will need to be done every time a new version of the chart is made.

The name of the branch is a reserved github name and needs to be exactly `gh-pages`. I believe it sets itself up automatically when pushing to this branch, otherwise the github pages setup is in the settings of the repo, near the bottom.

This whole thing will setup github to be the helm repository for squidex, so we can import it as a dependency without even checking out the squidex-docker repo. It might be adviseable to document that, let me know.

**Pull request merged**